### PR TITLE
Enhancement/#6-basic page to see all configured mappings

### PR DESCRIPTION
**Pull request by OtakuDigitChoc**

### CODE REVIEW

File: src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java [hunk 1]
```diff
@@ -5,26 +5,29 @@
...
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 import static org.springframework.web.bind.annotation.RequestMethod.POST;
 
 @RestController
-@RequestMapping("/api/dc-model-mapping")
+@RequestMapping("/api/dc-exporter")
```
Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java]: Why did you change the base path ?

File: src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java [hunk 2]
```diff
@@ -38,4 +41,10 @@
         }
         return new ResponseEntity<>(HttpStatus.CONFLICT);
     }
+
+    @RequestMapping(value = "/logs", method = RequestMethod.GET)
+    public @ResponseBody
+    List<DcModelMapping> getTags() {
```
Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java]: getTags ?

File: src/main/resources/public/jsx/Panel.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        {children}
+    </div>
+
+const PanelGroup = ({children}) =>
+    <div className="panel-group">
+        {children}
+    </div>
+
+const Panel = ({log}) =>
+    //TODO:replace "true" with sync result
```
Comment (bobeal) [inline src/main/resources/public/jsx/Panel.jsx]: Oh oh

File: src/main/resources/public/jsx/dashboard.jsx [hunk 4]
```diff
@@ -1,9 +1,41 @@
...
+                </ContainerPanel>
+            </div>
         )
     }
-})
+}
+
+const LogsList =  ({logs}) => {
+    for(var log of logs){
+        log.synchronizerAuditLog.date = new Date(log.synchronizerAuditLog.date).toUTCString()
```
Comment (bobeal) [inline src/main/resources/public/jsx/dashboard.jsx]: You could do it directly in the Panel component

File: src/main/java/org/ozwillo/dcexporter/service/DcModelMappingService.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+public class DcModelMappingService {
+
+    @Autowired
+    private DcModelMappingRepository dcModelMappingRepository;
+
+    @Autowired
+    private SynchronizerAuditLogRepository synchronizerAuditLogRepository;
+
+    public List<DcModelMapping> getAllAuditLogWithModel() {
+        List<DcModelMapping> dcModelMappingListWithLog = new ArrayList<DcModelMapping>();
```
Comment (bobeal) [inline src/main/java/org/ozwillo/dcexporter/service/DcModelMappingService.java]: Not really fan of this.

Would be better to keep model mappings and audit logs separated. For instance, first fetch the model mappings, then for each of them, fetch the audit log.

File: src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java (hunk 1 above)
Comment (OtakuDigitChoc) [inline src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java]: When I added the logs, I respected the naming of other mapping "api / ckan", "api / dc". It's not good ?

File: src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java (hunk 2 above)
Comment (OtakuDigitChoc) [inline src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java]: careless mistake

File: src/main/java/org/ozwillo/dcexporter/service/DcModelMappingService.java (hunk 5 above)
Comment (OtakuDigitChoc) [inline src/main/java/org/ozwillo/dcexporter/service/DcModelMappingService.java]: It's better that I create a new class that contains two objects "model mapping" and "audit log" ?
